main.go

```diff
@@ -149,20 +149,17 @@ func runJSONServer(ctx context.Context, grpcBind, jsonBind string) {
 }
 
 func main() {
+	kingpin.Version(version.ShortInfo())
+	grpcBind := kingpin.Flag("grpcBind", "GRPC bind address and port").Default("127.0.0.1:9911").String()
```

Comment:
`.Envar("QANAPI_BIND")` for compatibility. The same for other flags.

Reply:
Added.